### PR TITLE
Update go version - github workflows

### PR DESCRIPTION
**Pull request by ssd04**

### CODE REVIEW

File: .github/workflows/pr-build.yaml [hunk 1]
```diff
@@ -2,19 +2,19 @@ name: Go Build
...
   build:
     name: Build
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go 1.x
+      - name: Set up Go 1.17.6
         uses: actions/setup-go@v2
         with:
-          go-version: ^1.15.6
+          go-version: ^1.17.6
```
Comment (bogdan-rosianu) [inline .github/workflows/pr-build.yaml:17]: remove `^` as current libs will fail if go1.19 is used for example

File: .github/workflows/pr-tests.yaml [hunk 2]
```diff
@@ -2,19 +2,19 @@ name: Tests
...
   test:
     name: Unit
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go 1.x
+      - name: Set up Go 1.17.6
         uses: actions/setup-go@v2
         with:
-          go-version: ^1.15.6
+          go-version: ^1.17.6
```
Comment (bogdan-rosianu) [inline .github/workflows/pr-tests.yaml:17]: remove `^`

File: .github/workflows/pr-build.yaml (hunk 1 above)
Comment (ssd04) [reply .github/workflows/pr-build.yaml]: removed

File: .github/workflows/pr-tests.yaml (hunk 2 above)
Comment (ssd04) [reply .github/workflows/pr-tests.yaml]: removed

**Review by bogdan-rosianu — APPROVED** [review]

**Review by schimih — APPROVED** [review]

**Pull request merged**